.github/workflows/build-package.yml

```diff
@@ -0,0 +1,132 @@
+name: "Build Package Workflow"
+env:
+  DOCKER_COMPOSE_VERSION: 1.23.0
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: 0 2 * * *
```

Comment:
do we need a cron here?

Reply:
As we discussed at the beginning, if goharbor/harbor upstream can trigger harbor-arm build, we don’t need timed tasks, but no, we need timed tasks to achieve the same goal